PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/webapp/action/CreateWebAppAction.java

```diff
@@ -146,7 +146,7 @@ private void deploy(final WebApp webapp, final Path application, final Project p
     private void refreshAzureExplorer(WebApp app) {
         AzureTaskManager.getInstance().runLater(() -> {
             if (AzureUIRefreshCore.listeners != null) {
-                AzureUIRefreshCore.execute(new AzureUIRefreshEvent(AzureUIRefreshEvent.EventType.REFRESH, app));
+                AzureUIRefreshCore.execute(new AzureUIRefreshEvent(AzureUIRefreshEvent.EventType.REFRESH, null));
```

Comment:
We may need to refactor `createListener` in `WebAppModule` for this issue

Reply:
let's do it in the future.